.githooks/pre-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Don't push unless the local branch contains the head of the remote
+# origin/develop branch.
+
+set -e
+
+# Get the head of the branch being pushed.
+#
+# Local SHAs are passed to stdin, in the second column. See pre-push.sample
+read _ local_sha _ _ 
+
+# Make sure we have the latest version of develop, locally
+git remote update > /dev/null
+
+# Check that pushee branch contains develop head SHA
```

Comment:
```suggestion
# Check that pushed branch contains develop head SHA
```

Reply:
Thanks. "pushee" was intentional, but it's too unconventional to be clear.